service/lambdaFunction/index.js

```diff
@@ -49,7 +51,9 @@ const regularRun = async () => {
             await context.tracing.stop({
                 path: path.join(__dirname, '..', '..', 'tmp', 'trace.zip'),
             });
-            await capture.stop();
+            if (process.env.IS_RECORD === 'to_record') {
```

Comment:
If the value of process.env.IS_RECORD is something that the user inserts, the best thing to do when comparing it to a string is to first have it all in lowercase (something like if (process.env.IS_RECORD.toLowerCase() === 'to_record').